Make dcdd shock capture optional for heat transfer (#1268)
chaos-polymtl/lethe: doc/source/parameters/cfd/multiphysics.rst

    set heat transfer                   = false
    set viscous dissipation             = false
    set buoyancy force                  = false

>     set DCDD shock capture              = false

Reviewer: I think this parameter would be better in the stabilization section and not the multiphysics sectoin.

Reviewer 2: Agreed

   * ``buoyancy force``: controls if the buoyancy force is taken into account in the Navier-Stokes equations. The buoyancy force is calculated using the Boussinesq approximation.

>    * ``DCDD shock capture``: The discontinuity-capturing directional dissipation method is used the stabilize the heat transfer equation. For more information, see Tezduyar, T. E. (2003). 

Reviewer 2: ```suggestion
   * ``DCDD shock capture``: Applies the discontinuity-capturing directional dissipation (DCDD) stabilization term on the heat transfer equation. For more information, see Tezduyar, T. E. (2003).
```

Reviewer 2: If you change the section, please don't forget to change the documentation accordingly.
